assets/src/adaptivity/scripting.ts

```diff
@@ -468,6 +468,29 @@ export const extractAllExpressionsFromText = (text: string): string[] => {
   return expressions;
 };
 
+export const formatNumber = (number: number) => {
+  const arrNumber = number.toString().split('.');
+  const containsDecimal = arrNumber.length > 1;
+  if (!containsDecimal) {
+    return number;
+  }
+  const decimalNumber = arrNumber[1];
+  const LeadingZerosInNumber = decimalNumber.toString().match(/\b0+/g);
+  let totalLeadingZerosInNumber = 0;
+  if (LeadingZerosInNumber?.length) {
+    totalLeadingZerosInNumber = LeadingZerosInNumber[0].length;
+  }
+
+  const modifiedNumber =
```

Comment:
The result should be as follows:

12.4567 - 12.46
11.88  - 11.88
0.0000126 - 0.000017 (previously it was rounding of to 0.00)
0.000120 - 0.00012